src/commands/createRepo/RemoteShortnameStep.ts

```diff
@@ -29,7 +29,11 @@ export class RemoteShortnameStep extends AzureWizardPromptStep<IStaticWebAppWiza
                     try {
                         await cpUtils.executeCommand(undefined, undefined, 'git', 'check-ref-format', '--branch', cpUtils.wrapArgInQuotes(value));
                     } catch (err) {
-                        return localize('notValid', '"{0}" is not a valid remote shortname.', value);
+                        if (/is not a valid branch name/g.test(parseError(err).message)) {
```

Comment:
I think this is the only command there is no equivalent for.  If the error that comes back doesn't match, I think we should ignore it since it's just name validation anyway and it'll fail later in the pipeline if it's really invalid.

Reply:
Hmm, I think it would be better to have it for a better user experience.  Getting through all of the prompts and having it fail after we create the GitHub remote would be pretty annoying.  And being able to see invalid names before entering them is a better experience.  I think the only reason it wouldn't work is if there was something stopping git from executing, such as being in a directory where they do not have access rights.  I think I'll change the executeCommand to run in the `cwd` though rather than the temp directory (what it does by default)